src/evict/evict_page.c

```diff
@@ -124,6 +123,7 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
      * Track how long forcible eviction took. Immediately increment the forcible eviction counter,
      * we might do an in-memory split and not an eviction, which skips the other statistics.
      */
+    time_start = 0;
```

Comment:
The same goes for this initialization it feels out of style.

Reply:
I disagree strongly on this one -- we're swapping out tests of `LF_ISSET(WT_EVICT_CALL_URGENT)` for tests of `time_start`, and I want it very, very clear how that works.

Reply:
If you look farther down, there's this:
```
    if (time_start != 0) {
        time_stop = __wt_clock(session);
        if (force_evict_hs)
            WT_STAT_CONN_INCR(session, cache_eviction_force_hs_success);
```
and `time_start` has nothing to do with `force_evict_hs`. This is the one chunk of this change I don't like where we ended up.

Reply:
Happy to leave it as is, I'm not fussed about this. 👍

Reply:
Yeah that `force_evict_hs` definitely feels out of place.